DataManagementSystem/DB/FileCatalogComponents/DirectoryMetadata.py

```diff
@@ -302,7 +314,7 @@ def getDirectoryMetaParameters(self, dpath, credDict, inherited=True, owndata=Tr
 
     return S_OK(metaDict)
 
-  def getDirectoryMetadata(self, path, credDict, inherited=True, owndata=True):
+  def getDirectoryMetadata(self, path, credDict, inherited=True, owndata=True, strip_suffix=False):
```

Comment:
It would be helpful to have a description under which circumestances strip_suffix true or false should be used

Reply:
```strip_suffix=true```  is to be used when a call is meant to be in a call chain which returns data to a user, otherwise it should be ``` false``` to maintain backward compatibility since that method is used internally as well.